ert3/storage/_storage.py

```diff
@@ -399,11 +430,14 @@ def _get_experiment_parameters(
         raise ert3.exceptions.StorageError(response.text)
     parameters: MutableMapping[str, List[str]] = {}
     for name in response.json():
-        key, val = name.split(".")
-        if key in parameters:
-            parameters[key].append(val)
+        if len(name.split(".")) == 1:
+            parameters[name] = []
         else:
-            parameters[key] = [val]
+            key, val = name.split(".")
+            if key in parameters:
```

Comment:
I know this is not you, but `defaultdict` is really nice for situations like this where you want to append to an initially empty list...